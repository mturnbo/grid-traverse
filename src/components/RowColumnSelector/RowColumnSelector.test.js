import React from 'react';
import { mount } from 'enzyme';
import RowColumnSelectorComponent from './index';

const wrapper = mount(<RowColumnSelectorComponent label="Test" selector="rows" size={10} />);

test('RowColumnSelector Component Renders', () => {
  expect(wrapper).toMatchSnapshot();
});
