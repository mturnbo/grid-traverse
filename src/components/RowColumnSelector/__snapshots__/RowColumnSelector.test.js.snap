// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowColumnSelector Component Renders 1`] = `
<RowColumnSelector
  label="Test"
  selector="rows"
  size={10}
>
  <div>
    <label>
      Test
    </label>
    <select>
      <option
        key="0"
        value={1}
      >
        1
      </option>
      <option
        key="1"
        value={2}
      >
        2
      </option>
      <option
        key="2"
        value={3}
      >
        3
      </option>
      <option
        key="3"
        value={4}
      >
        4
      </option>
      <option
        key="4"
        value={5}
      >
        5
      </option>
      <option
        key="5"
        value={6}
      >
        6
      </option>
      <option
        key="6"
        value={7}
      >
        7
      </option>
      <option
        key="7"
        value={8}
      >
        8
      </option>
      <option
        key="8"
        value={9}
      >
        9
      </option>
      <option
        key="9"
        value={10}
      >
        10
      </option>
    </select>
  </div>
</RowColumnSelector>
`;
